### ISSUE TITLE
Accessibility snapshots highlight colours change when the view is edited

### ISSUE THREAD

**Issue by colinmarsch** [cashapp/paparazzi#348]
After updating a `View` class that has an accessibility snapshot created for it, the accessibility Paparazzi tests for it fail as the colours used to highlight the children in the image change.

Ideally, the accessibility test would still generate a snapshot with the same colours used previously to avoid needing to replace the existing images.
